serve/graph/model/block.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+)
+
+type Block struct {
+	b *types.Block
+}
+
+func NewBlock(b *types.Block) *Block {
+	return &Block{
+		b: b,
+	}
+}
+
+func (b *Block) ID() string {
```

Comment:
Are the `ID` methods here and in transaction used?

Reply:
Are used internally for the graphQL library for caching and other functionalities that improve performance.